ros2doctor/ros2doctor/command/doctor.py

```diff
@@ -49,7 +49,7 @@ def main(self, *, parser, args):
         failed_cats, fail, total = run_checks(include_warnings=args.include_warnings)
         if fail:
             print('\n%d/%d checks failed\n' % (fail, total))
-            print('Failed modules are ', *failed_cats)
+            print('Failed modules: ', *failed_cats)
```

Comment:
The space after the colcon should be removed to avoid a double space before the first name.